automation.py

```diff
@@ -133,6 +133,9 @@ def _validate(self, config_dictionary):
 
     def run(self, library_filter=""):
         try:
+            version = self.cmdProvider.run(["git", "log", "-1"], shell=False, clean_return=True).stdout.decode().strip()
```

Comment:
Won't this output an entire comit message?  Maybe you mean `git log -1 --oneline`?

Reply:
`--oneline` is exactly what i wanted but only tested it on commits containing a single line 😅 oops